base_changeset/static/src/js/list_backend.js

```diff
@@ -0,0 +1,159 @@
+odoo.define("base_changeset.list_backend", function(require) {
+    "use strict";
+
+    var ListRenderer = require("web.ListRenderer");
+    var ListController = require("web.ListController");
+    var core = require("web.core");
+    var qweb = core.qweb;
+
+    ListController.include({
+        start: function() {
+            return this._super
+                .apply(this, arguments)
+                .then(this._updateChangeset.bind(this));
+        },
+
+        update: function() {
+            var self = this;
+            var res = this._super.apply(this, arguments);
+            res.then(function() {
+                self._updateChangeset();
+            });
+            return res;
+        },
+
+        _updateChangeset: function() {
+            var self = this;
+            var state = this.model.get(this.handle);
+            this.model.getChangeset(state.model, false).then(function(changeset) {
+                self.renderer.renderChangesetPopovers(changeset);
+            });
+        },
+
+        applyChange: function(id) {
+            this.model.applyChange(id).then(this.reload.bind(this));
+        },
+
+        rejectChange: function(id) {
+            this.model.rejectChange(id).then(this.reload.bind(this));
+        },
+    });
+
+    ListRenderer.include({
+        /* eslint-disable no-unused-vars */
+        init: function(parent, state, params) {
```

Comment:
You can remove the eslint exception and remove the parameters and it will work fine.